core/src/test/java/kafka/test/ClusterInstance.java

```diff
@@ -145,4 +152,11 @@ default Admin createAdminClient() {
     void startBroker(int brokerId);
 
     void waitForReadyBrokers() throws InterruptedException;
+
+    default Iterable<GroupProtocol> supportedGroupProtocol() {
```

Comment:
Being a shared method, `Set` is more suitable return type. That makes sure no duplicate values in the returned collection